tests/unit/test_utils.py

```diff
@@ -273,7 +273,7 @@ def test_rmtree_skips_nonexistent_directory() -> None:
     rmtree.__wrapped__("nonexistent-subdir")  # type: ignore[attr-defined]
 
 
-class Failer:
```

Comment:
This isn't a typo fix, it is a class name. I don't think there should be a blanket prohibition on using "made up" terms for function or class names.

I'm broadly indifferent about changing the name of the class[^1], but I'd prefer it (a) to be in a separate PR, and (b) have a better justification than "it doesn't spellcheck".

[^1]: In general, I see no benefit in churn like this unless it fixes something.

Reply:
I can 
- make `codespell` completely ignore any `failer` word, just say so.
- send that commit as a separate PR with an explanation along following lines which would be weak without codespell argument:

I did go extra mile to come up with a better fitting class name here, indeed though motivated by codespell since otherwise only people fixing some tests there would need to read the code to realize what `Failer` does, since e.g.

`Failer(duration=1)` -- does it fail for a second and then not? or fails after a second?

With the name `FailAfter(duration=1)` it becomes clear and codespell is happy.

Reply:
ok, I left Failer unchanged